infrastructure/staging/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "region" {
+  default     = "us-east-1"
+  description = "The AWS Region"
+}
+
+variable "saml_role" {
+  description = "Name of the role that has access to TFState"
+}
+
+variable "app" {
+  description = "Name of the application"
+}
+
+variable "env" {
+  description = "The environment for this deployment"
+  default     = "staging"
+}
+
+variable "team" {
+  description = "The team responsible"
+}
+
+variable "customer" {
+  description = "The customer for this application"
+}
```

Comment:
This will be obvious based on the account, so this is not necessary.

Reply:
Removed